ui/api.py

```diff
@@ -33,7 +33,7 @@ def process_dropbox_data(dropbox_upload_data):
         with transaction.atomic():
             video = models.Video.objects.create(
                 source_url=dropbox_link["link"],
-                title=dropbox_link["name"],
+                title=dropbox_link["name"][:250],
```

Comment:
Could you just get this from the model itself? `models.Video.title.max_length` or something like that. Not sure if that can be done but worth a look

Reply:
Changed the code to get the length from the model metadata